qabelbox/src/main/java/de/qabel/qabelbox/dagger/modules/MainActivityModule.java

```diff
@@ -49,7 +49,11 @@ public MainActivity provideMainActivity() {
         } else {
             String keyId = sharedPreferences.getLastActiveIdentityKey();
             try {
-                return identityRepository.find(keyId);
+                if(keyId != null){
+                    return identityRepository.find(keyId);
+                }else {
+                    throw new PersistenceException("Not last active identity found");
```

Comment:
Since this exception is catched, the message is never logged. Add a Log.w about that.